### PR TITLE
Sponsor user service

### PR DESCRIPTION
**Pull request by rperez89**

### CODE REVIEW

File: app/src/services/sponsorUser.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      error: response.errorMessage,
+    }
+  } catch (err) {
+    console.error(err)
+    return { error: err }
+  }
+}
+
+function sponsorKeyAndSig(account) {
+  const privateKey = 'OMITTED FOR GIT'
```
Comment (fabriziovigevani) [inline app/src/services/sponsorUser.js:47]: This should be the saved SECRET right?

File: app/src/constants.js [hunk 2]
```diff
@@ -2,7 +2,8 @@ export const BRIGHTID_CONTEXT = '1hive'
 
 export const NODE_URL = 'http:%2f%2fnode.brightid.org'
 
-export const BRIGHTID_VERIFICATION_ENDPOINT =
-  'http://node.brightid.org/brightid/v4/verifications'
+export const BRIGHT_ID_ENDPOINT = 'http://node.brightid.org/brightid/v4'
+export const BRIGHTID_VERIFICATION_ENDPOINT = `${BRIGHT_ID_ENDPOINT}/verifications`
+export const BRIGHTID_SUBSCRIPTION_ENDPOINT = `${BRIGHT_ID_ENDPOINT}/operations`
```
Comment (fabriziovigevani) [inline app/src/constants.js:7]: Wondering if we should move this ones to the  `endpoints` file once we merge #13 ?

File: app/src/services/sponsorUser.js (hunk 1 above)
Comment (rperez89) [reply app/src/services/sponsorUser.js:47]: ahh yes totally forgot to remove that haha

File: app/src/constants.js (hunk 2 above)
Comment (rperez89) [reply app/src/constants.js:7]: yes totally

**Pull request merged**